usr.sbin/traceroute/tests/traceroute4_test.sh

```
@@ -0,0 +1,128 @@
+# SPDX-License-Identifier: ISC
+#
+# Copyright (c) 2025 Lexi Winter
+#
+# Permission to use, copy, modify, and distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+. $(atf_get_srcdir)/../../sys/common/vnet.subr
+
+TRACEROUTE="traceroute -w 1 -q 1 -m 5"
+
+setup_network()
+{
+	# Create 3 jails: one to be the source host, one to be the router,
+	# and one to be the destination host.
+
+	vnet_init
+
+	# src jail
+	epsrc=$(vnet_mkepair)
+	vnet_mkjail trsrc ${epsrc}a
+
+	# dst jail
+	epdst=$(vnet_mkepair)
+	vnet_mkjail trdst ${epdst}a
+
+	# router jail
+	vnet_mkjail trrtr ${epsrc}b ${epdst}b
+
+	# Configure IPv4 addresses and routes on each jail
+	jexec trsrc ifconfig ${epsrc}a inet 192.0.2.2/31 up
+	jexec trrtr ifconfig ${epsrc}b inet 192.0.2.3/31 up
+	jexec trsrc route add -inet 192.0.2.0/24 192.0.2.3
+
+	jexec trdst ifconfig ${epdst}a inet 192.0.2.4/31 up
+	jexec trrtr ifconfig ${epdst}b inet 192.0.2.5/31 up
+	jexec trdst route add -inet 192.0.2.0/24 192.0.2.5
+
+	jexec trrtr sysctl -w net.inet.ip.forwarding=1
```

The `-w` option is deprecated, and it is silently ignored.

---

fixed